ports/atmel-samd/common-hal/pulseio/PWMOut.c

```diff
@@ -425,7 +427,7 @@ void common_hal_pulseio_pwmout_set_frequency(pulseio_pwmout_obj_t* self,
         tc->COUNT16.CC[0].reg = new_top;
         #endif
         #ifdef SAMD51
-        while (tc->COUNT16.SYNCBUSY.reg != 0) {
+        while (tc->COUNT16.SYNCBUSY.reg != 0) {}
             /* Wait for sync */
         }
```

Comment:
Leftover closing bracket from previous `while SYNCBUSY`. Fairly certain this is causing the Travis fails.

Reply:
Thanks for catching that! I did an edit after a test compile, and didn't repeat it :(